service/src/test/java/uk/gov/hmcts/reform/fpl/testbeans/TestFeeConfig.java

```diff
@@ -59,6 +59,7 @@ public class TestFeeConfig {
     public static final String RECOVERY_ORDER_KEYWORD = "Recovery";
     public static final String SECURE_ACCOMMODATION_ENG_KEYWORD = "SecureAccommEngland";
     public static final String SECURE_ACCOMMODATION_WALES_KEYWORD = "SecureAccommWales";
+    public static final String CONTACT_WITH_CHILD_IN_CARE_KEYWORD = "CIC";
```

Comment:
Well, this is for unit test only, so should be fine.
I wonder the keyword should be "WXY" or "ContactChild"

FPL FeeType mapping is configured in application.yaml

Fee code table:
https://tools.hmcts.net/confluence/display/RP/Fee+Register+-+Live+Fees+CSV